src/app/components/SubjectHeading/SubjectHeadingShow.jsx

```diff
@@ -1,12 +1,11 @@
-import React from 'react';
-import PropTypes from 'prop-types';
 import axios from 'axios';
```

Comment:
Committed Prettied file. No Functional Changes were done to this file. Soley format.